### PR TITLE
Possible fix for #31.

### PR DESCRIPTION
**Pull request by mamoll**
Completely untested (it probably won't even compile!).

### CODE REVIEW

**Comment by isucan** [comment]
This is a correct fix. +1

File: ompl/ompl_interface/src/parameterization/model_based_state_space.cpp [hunk 1]
```diff
@@ -147,6 +147,19 @@ double ompl_interface::ModelBasedStateSpace::getMaximumExtent() const
...
 }
 
+double ompl_interface::getMeasure() const
+{
+    double m = 1.0;
+    for (std::size_t i = 0 ; i < spec_.joint_bounds_.size() ; ++i)
+    {
+        const JointModel::Bounds& bounds = *spec_.joint_bounds_[i];
+        for (std::size_t j = 0 ; j < bounds.size() ; ++j)
+            m *= bounds[j].max_position_ - sbounds[j].min_position_;
```
Comment (isucan) [inline ompl/ompl_interface/src/parameterization/model_based_state_space.cpp]: Sbounds is probably a typo

**Comment by davetcoleman** [comment]
I will test today.

**Comment by davetcoleman** [comment]
New PR: https://github.com/ros-planning/moveit_planners/pull/33

**Pull request closed without merging**